app/src/store/data.ts

```diff
@@ -295,18 +298,24 @@ export default function useDataStore() {
    * @notice helper function to filter approved grants
    *
    * @param grants Grant[]
+   * @param approvedList []
```

Comment:
The `approvedList` entry should be deleted and `grants` is missing the description string